Makefile

```diff
@@ -27,6 +28,14 @@ all: build
 build:
 	GOBIN=$(BINDIR) $(GO) install $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' k8s.io/helm/cmd/...
 
+.PHONY: containerized-build
+containerized-build: check-docker
+	docker run \
+		-v $(shell pwd):/go/src/k8s.io/helm \
```

Comment:
Can we use `$(CURDIR)` instead? It's a [GNU Make builtin](https://www.gnu.org/software/make/manual/html_node/Quick-Reference.html).

Reply:
Yes, can do!